adapter.go

```diff
@@ -264,6 +267,18 @@ func TurnOffAutoMigrate(db *gorm.DB) *gorm.DB {
 }
 
 func NewAdapterByDBWithCustomTable(db *gorm.DB, t interface{}, tableName ...string) (*Adapter, error) {
+
+	r := reflect.ValueOf(t).Type()
+	if r.Kind() == reflect.Ptr {
+		r = r.Elem()
+	}
+	for _, field := range []string{"ID", "Ptype", "V0", "V1", "V2", "V3", "V4", "V5", "V6", "V7"} {
+		if _, ok := r.FieldByName(field); !ok {
+			panic(fmt.Sprintf("The custom table has no column named `%s`", field))
+			//return nil, errors.New(fmt.Sprintf("The custom table has no column named `%s`", field))
+		}
+	}
+
```

Comment:
Don't have to use reflect to check here, it's poor performance, and the loss is not worth it